tests/packagedcode/test_chef.py

```diff
@@ -46,7 +50,8 @@ def test_parse_from_rb(self):
         test_file = self.get_test_loc('chef/basic/metadata.rb')
         expected_file = self.get_test_loc('chef/basic/metadata.rb.expected')
         self.check_package(chef.parse(test_file), expected_file, regen=False)
-
+    
+    @skipIf(not py3, 'Python 3 only') # data are different for both Python And data generated by assigning regen = false 
```

Comment:
Add this whole comment `data are different for both Python And data generated by assigning regen = false ` to the skipif e.g.:
```
@pytext.xmarlk.skipif(not py3, reason='Python 3 only, data are different for both Python And data generated by assigning regen = false')
```

BUT why is this the case? that does not sound right